src/components/DeleteMovie/DeleteMovie.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+import Modal from "../Modal/Modal";
+import classes from "./DeleteMovie.module.css";
+
+const DeleteMovie: React.FC = () => {
+  const [modal, setModal] = useState(false);
+
+  const toggle = () => setModal(!modal);
+
```

Comment:
I see this state portion is a Boolean, but you named it `modal`. In your code, when I read word `modal` I think about a modal frame instance and do not expect it to be a Boolean.
More appropriate name for Boolean would start with `is`, for example: `isModalOpen`

Reply:
Done

Reply:
Resolved ✅